docs/TuxOwneeClass.md

```diff
@@ -0,0 +1,250 @@
+# Tux/React.OwneeClass
+
+## Table of Contents
+<ol>
+  <li><a href="#Premise">Premise</a></li>
+  <li><a href="#Implementation">Implementation</a>
+    <ol style="list-style-type:upper-alpha">
+      <li><a href="#Requiring-OwneeClass">Requiring OwneeClass</a></li>
+      <li><a href="#Creating-an-OwneeClass-Component">Creating an OwneeClass Component</a>
+        <ol style="list-style-type:upper-roman">
+          <li><a href="#Creating-an-OwneeClass-Component-Example">Creating an OwneeClass Component Example</a></li>
+          <li><a href="#React-createOwneeClass">METHOD: React.createOwneeClass</a></li>
+          <li><a href="#createOwneeClass-owneeClassProps">OBJECT: owneeClassProps</a>
+            <ol style="list-style-type:lower-alpha">
+              <li><a href="#owneeClassProps-nearestOwnerProps">PROPERTY: owneeClassProps.nearestOwnerProps</a></li>
+              <li><a href="#owneeClassProps-nearestOwnerPropTypes">OBJECT: owneeClassProps.nearestOwnerPropTypes</a></li>
+              <li><a href="#owneeClassProps-anyPropTypes">OBJECT: owneeClassProps.anyPropTypes</a></li>
+            </ol>
+          </li>
+        </ol>
+      </li>
+    </ol>
+  </li>
+  <li><a href="#TuxOwneeClass-Guidelines">TuxOwneeClass Guidelines</a>
+  <li><a href="#TuxOwneeClass-Complete-Example">TuxOwneeClass Complete Example</a>
+</ol>
+## <a id="Premise"></a>Premise [#](#Premise)
+>The `OwneeClass` is an opinionated [React](http://facebook.github.io/react/) class that is not designed to manage any of its own state other than, in some cases, necessary state to implement two-way data-binding for input validation. Instead the `OwneeClass` has access to  properties and methods that are kept on its nearest `OwnerClass` componet. The `OwnerClass` shares these properties and methods with the `OwneeClass` without having to cascade those properties down as per the usual `React` workflow.
```

Comment:
Correct spelling of component.

Change second sentence to: Instead the `OwneeClass` uses dynamic properties passed down by `props` as per the normal `React` workflow, and static properties and methods shared automatically by the nearest parent `OwnerClass`  component.